Modules/unicodedata.c

```diff
@@ -19,7 +19,7 @@
 #include "Python.h"
 #include "pycore_object.h"        // _PyObject_VisitType()
 #include "pycore_ucnhash.h"       // _PyUnicode_Name_CAPI
-#include "pycore_unicodeobject.h"   // _PyUnicode_IsXidStart, _PyUnicode_IsXidContinue
+#include "pycore_unicodectype.h"   // _PyUnicode_IsXidStart, _PyUnicode_IsXidContinue
```

Comment:
One function per include is enough:

```suggestion
#include "pycore_unicodectype.h"   // _PyUnicode_IsXidStart()
```